DataFormats/FEDRawData/src/classes_def.xml

```diff
@@ -24,4 +27,5 @@
  </class>
  <class name="edm::Wrapper<FEDRawDataCollection>" splitLevel="0"/>
  <class name="edm::RefProd<FEDRawDataCollection>"/>
+ <class name="edm::Wrapper<BSTRecord>" splitLevel="0"/>
```

Comment:
I would naively think that splitLevel applies to sub-branches (BSTRecord) as well.
If it does, then this is a functional (not a no-op) setting.
@Dr15Jones please clarify, thank you.